[2024-05-21T14:39:57.813+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-21T14:39:57.835+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: statistical_calculations_dag.generate_random_numbers scheduled__2022-12-07T00:00:00+00:00 [queued]>
[2024-05-21T14:39:57.845+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: statistical_calculations_dag.generate_random_numbers scheduled__2022-12-07T00:00:00+00:00 [queued]>
[2024-05-21T14:39:57.847+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-05-21T14:39:57.862+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): generate_random_numbers> on 2022-12-07 00:00:00+00:00
[2024-05-21T14:39:57.869+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=8223) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-21T14:39:57.870+0000] {standard_task_runner.py:63} INFO - Started process 8229 to run task
[2024-05-21T14:39:57.870+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'statistical_calculations_dag', 'generate_random_numbers', 'scheduled__2022-12-07T00:00:00+00:00', '--job-id', '2774', '--raw', '--subdir', 'DAGS_FOLDER/second_dag.py', '--cfg-path', '/tmp/tmp6yhwfnsw']
[2024-05-21T14:39:57.871+0000] {standard_task_runner.py:91} INFO - Job 2774: Subtask generate_random_numbers
[2024-05-21T14:39:57.927+0000] {task_command.py:426} INFO - Running <TaskInstance: statistical_calculations_dag.generate_random_numbers scheduled__2022-12-07T00:00:00+00:00 [running]> on host c420abfdc210
[2024-05-21T14:39:58.038+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='test' AIRFLOW_CTX_DAG_ID='statistical_calculations_dag' AIRFLOW_CTX_TASK_ID='generate_random_numbers' AIRFLOW_CTX_EXECUTION_DATE='2022-12-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-12-07T00:00:00+00:00'
[2024-05-21T14:39:58.039+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-21T14:39:58.066+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-21T14:39:58.070+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-21T14:39:58.080+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=statistical_calculations_dag, task_id=generate_random_numbers, run_id=scheduled__2022-12-07T00:00:00+00:00, execution_date=20221207T000000, start_date=20240521T143957, end_date=20240521T143958
[2024-05-21T14:39:58.125+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-21T14:39:58.135+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
